includes/admin/class-wc-stripe-rest-payment-intents-controller.php

```
@@ -0,0 +1,197 @@
+		return $this->rest_args;
+	}
+
+	/**
+	 * Given an incoming REST request, build and return an array of query parameters to be appended to Stripe API request URL.
+	 *
+	 * @param WP_REST_Request<array<string, mixed>> $request An incoming REST request.
+	 *
+	 * @return array
+	 */
+	public function build_http_query_array_from_request( $request ): array {
+		$route_args = $this->get_payment_intents_route_args();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$route_args'. (undefined)

(UnusedLocalVariable)